rest/data/aliases.go

```diff
@@ -25,7 +27,11 @@ func FindProjectAliases(projectId, repoId string, aliasesToAdd []restModel.APIPr
 		}
 	}
 	if projectId != "" {
-		aliases, err = model.FindAliasesForProjectFromDb(projectId)
+		if includeProjectConfig {
```

Comment:
I believe the order goes project -> repo -> config, from FindMergedProjectRef (if so we should add to https://github.com/evergreen-ci/evergreen/wiki/Project-and-Distro-Settings#version-control). In which case we shouldn't be merging with projectRef aliases until _after_ the repo. Perhaps FindAliasesMergedWithProjectConfig can be refactored to take in the project/repo aliases list instead of querying for it?

Reply:
Done, and updated docs: https://github.com/evergreen-ci/evergreen/wiki/Project-and-Distro-Settings#version-control